packages/snjs/test/lib/model_tests/notes_tags.test.js

```diff
@@ -0,0 +1,910 @@
+import { PayloadSource, CreateMaxPayloadFromAnyObject, NotesDisplayCriteria } from '@Lib/index';
+import { ContentType, SNNote, SNPredicate } from '@Lib/models';
+import * as Factory from '../../factory';
+
+describe('notes and tags', () => {
```

Comment:
Why is git treating this as a new file, but other tests get recognized as a move of an existing file? Makes it hard to track changes.

Reply:
Not sure what should the issue be, maybe it's related with GitHub? I can still see changes from the the previous file location, using VSCode and/or via `git diff`. The file was move [in this commit](https://github.com/standardnotes/snjs/pull/357/commits/03e22a4b2d05a6b27a3aa259c85dd1131f24081c#diff-62e819e9773db2ee27dfc5f3fad0f3986502225647f9be9d41289089c7d456ee) and should keep history.